pkg/drivers/kic/oci/oci.go

```diff
@@ -76,7 +76,7 @@ func DeleteContainer(ociBin string, name string) error {
 
 	_, err := ContainerStatus(ociBin, name)
 	if err != nil {
-		glog.Errorf("%s daemon seems to be stuck. Please try restarting your %s. Will try to delete anyways: %v", ociBin, ociBin, err)
+		glog.Infof("%s daemon seems to be stuck. Will try to delete anyways: %v", ociBin, err)
```

Comment:
this is a good opportunity for a refactor,  this log is misleading because it could be NOT the case that the docker is stuck.
the only case when docker is stuck is when "docker inspect" cmomand times out ( for safty we could wait up to 13 second but if it is not returning 2 seconds it is totally stuck)
in that case we need to return an error type called DockerStuckNeedsRestart

and we should return that error to the Exit usage
in other cases we should log it as warnning
I recommend that we make ContainerStatus to return better error type

Reply:
aslo I recommend using warrningf instead of info. warrning would not show up to the user

Reply:
sounds good, we now log it as a warning if docker inspect takes >10 seconds to respond